### PR TITLE
Fix spell popover close actions

### PR DESCRIPTION
**Pull request by jazeee**
Summary
===
The popover close buttons no longer work. This may be related to changes made by roll20 exposing a bug in the Epic character sheet.
Our bug: We forgot to add `repeating_${spellDomain}:` to the click listener.

> The Detail window does not close either through the "Close" button or clicking the "X".
image

Changes
===
- Update click listener to include `repeating_${spellDomain}:`

### CODE REVIEW

File: ui/scripts/inlineHtml/inlineScripts.js [hunk 1]
```diff
@@ -495,7 +495,7 @@ function setupSpellNotes(spellDomain) {
     });
   });
 
-  on('clicked:closepopoverspellnotes', closeAllSpellNotes);
+  on(`clicked:repeating_${spellDomain}:closepopoverspellnotes`, closeAllSpellNotes);
```
Comment (jazeee) [inline ui/scripts/inlineHtml/inlineScripts.js:498]: I think we had a hidden bug in our code. I suspect that Roll20 tightened up their code, which broke this incorrect listener.
It should have had a repeating selector.

Comment (johnlamping) [reply ui/scripts/inlineHtml/inlineScripts.js:498]: How did you find that so fast?

**Review by johnlamping — APPROVED** [review]
Thanks!

File: ui/scripts/inlineHtml/inlineScripts.js (hunk 1 above)
Comment (jazeee) [reply ui/scripts/inlineHtml/inlineScripts.js:498]: I ended up testing several button combinations, and eventually found this was missing.
A bit of trial and error, really.

**Pull request merged**